packages/fluentui/react-northstar-prototypes/src/prototypes/VirtualizedStickyTree/VirtualStickyTree.tsx

```diff
@@ -155,12 +197,10 @@ export const VirtualStickyTree: ComponentWithAs<'div', VirtualStickyTreeProps> =
       };
 
       if (isOverlappingWithSticky(id)) {
-        // known issue:
-        // this scroll cannot guarantee non-sticky items' visibility. It would work when sticky titles are relatively small compare to non-sticky items.
-        listRef.current.scrollToItem(visibleItemIds.indexOf(id), 'center'); // scroll to item
+        scrollToMakeItemVisible(style.top as number);
```

Comment:
is `style.top` always set for all items ?

Reply:
Good question. Yes it is: https://github.com/bvaughn/react-window/blob/d80bef25fe706d0c73fc801674c086f681811190/src/createListComponent.js#L475

Reply:
Since this is reference documentation on our side, I think you should handle `style.top` as `undefined` case in the function. `0` should be a pretty appropriate default

`undefined/2 === NaN` in JS, and `Math.max(0, NaN) === NaN` 

We don't want to be responsible for weird things happening in this boundary case

Reply:
Good point. I'm surprised `Math.max(0, NaN)` is NaN 😮 
what about 
```suggestion
        scrollToMakeItemVisible(style.top || 0);
```

Reply:
Either there or in the `scrollToMakeItemVisible` function as a guard, both work